.github/workflows/check-translations-and-deserialize.yml

```diff
@@ -75,11 +75,15 @@ jobs:
           team-reviewers: developer-enablement
 
       - name: Remove Completed Batches From Queue
-        if: steps.fetch-deserialize.outputs.batchesToDeserialize != 0
+        if: steps.fetch-deserialize.outputs.successfulTranslations != 0
```

Comment:
We probably also want to fail this step if we were unable to create a pull request (the previous step). Otherwise we might risk removing something from the DB and _not_ having the files committed to the repo.

Reply:
I misspoke when I mentioned this to you yesterday 🙈 

I misunderstood how this would execute, but now I've got that sorted in my head. This step won't run if the previous step fails. We can actually see this in the logs of our currently failing action, so this should be good as is.

when the pr doesn't run, this won't run because they share a condition. if the commit step fails, or the pr step fails, the workflow doesn't reach this step and it won't run.

Reply:
^ that whole thing is correct, but im wrong again because I forgot what we talked about 🙈 

we wanted to make sure that if we accidentally change the workflow, that the cleanup step is always dependent on translations being completed, and the PR running successfully. we want to avoid the possibility of the PR not running, and cleaning up records which we might introduce with a typo in the condition, etc.